modules/assim.sequential/DESCRIPTION

```diff
@@ -64,4 +64,4 @@ Suggests:
 License: BSD_3_clause + file LICENSE
 Copyright: Authors
 Encoding: UTF-8
-RoxygenNote: 7.3.2
+RoxygenNote: 7.3.1
```

Comment:
Roxygen version is wrong, please fix and make sure you have the correct version installed

Reply:
addressed in [0cf995a](https://github.com/PecanProject/pecan/pull/3321/commits/0cf995a432807c2a46b308a5d5a027fcfc3e0871) commit